README.md

```diff
@@ -137,571 +135,54 @@ zplug denysdovhan/spaceship-zsh-theme, use:spaceship.zsh, from:github, as:theme
 
 If you have problems with approches above, follow these instructions:
 
-- Download the theme [here](https://raw.githubusercontent.com/denysdovhan/spaceship-zsh-theme/master/spaceship.zsh)
-- Symlink `spaceship.zsh` to somewhere in `$fpath` as `prompt_spaceship_setup`.
+* Clone this repo `git clone https://github.com/denysdovhan/spaceship-zsh-theme.git`
+* Symlink `spaceship.zsh` to somewhere in [`$fpath`](http://www.refining-linux.org/archives/46/ZSH-Gem-12-Autoloading-functions/) as `prompt_spaceship_setup`.
+* Initialize prompt system and choose `spaceship`.
+
+#### Example
```

Comment:
Do you mean adding a section on how to install it with prezto? But the theme is not even added yet in prezto-contrib, https://github.com/sorin-ionescu/prezto/issues/1504

Reply:
Most Zsh frameworks and Zsh plugin managers except prezto are specified in installation methods. `spaceship` wasn't compatible with prezto till 3.0, So users might assume the same. 

It would be great if we at least specify how to install `spaceship` on prezto.

Reply:
I don't use `prezto`. Can you add a section about installing for its users?

Reply:
prezto follows vanilla Zsh prompt setup, Manual installation methods will get the job done. But we just haven't specified it anywhere.